source/opcode.cpp

```diff
@@ -719,3 +719,14 @@ std::vector<uint32_t> spvOpcodeMemorySemanticsOperandIndices(SpvOp opcode) {
       return {};
   }
 }
+
+bool spvOpcodeIsAccessChain(SpvOp opcode) {
+  switch (opcode) {
+    case SpvOpAccessChain:
+    case SpvOpInBoundsAccessChain:
+    case SpvOpPtrAccessChain:
+      return true;
+    default:
+      return false;
+  }
+}
```

Comment:
It might be better to make it a separate PR since it changes some code outside of the spirv-fuzz tool. I'm not sure whether this is critical, though. @paulthomson, what do you think?

Reply:
I agree. We prefer to let the spirv-opt folks approve changes in their code.

Reply:
But it looks like a good change so I am guess they will approve.

Reply:
Done.